Python/ceval.c

```diff
@@ -4849,7 +4849,15 @@ import_all_from(PyObject *locals, PyObject *v)
                 PyErr_Clear();
             break;
         }
-        if (skip_leading_underscores && PyUnicode_Check(name)) {
+        if (!PyUnicode_Check(name)) {
+            PyErr_Format(PyExc_TypeError,
+                         "Item in __all__ must be str, not %.100s",
```

Comment:
Wouldn't be better to include the module name as in Python version?

If `skip_leading_underscores` is true, the source of names is `__dict__`, not `__all__`.

Reply:
Of course it would be, but I hesitate. Is `v` always a modules and I can use `PyModule_GetName` then? Or using `__name__`? `__name__` could be altered.